runtime/src/iree/task/api.c

```diff
@@ -98,11 +98,15 @@ IREE_FLAG(
     "'current' will inherit the node of the calling thread.");
 
 IREE_FLAG(
-    int32_t, task_topology_max_group_count, 8,
+    int32_t, task_topology_max_group_count, 64,
     "Sets a maximum value on the worker count that can be automatically\n"
     "detected and used when --task_topology_group_count=0 and is ignored\n"
     "otherwise.");
```

Comment:
Worth running CI benchmarks to verify that this default change doesn't unexpectedly regress Android performance or something? (Not sure if this is set explicitly there)

Reply:
I hope they aren't relying on default flag values - what's the magic to run them?

Reply:
I think just add labels to the PR, e.g.
![image](https://github.com/openxla/iree/assets/4010439/d3d49541-d286-40b0-b4da-b97fa5adf4ab)

I see a few spots setting the flag, at least: https://github.com/openxla/iree/blob/2529fb37c76c89c0306a4bb45f8bdcdf8baeaf16/build_tools/python/benchmark_suites/iree/module_execution_configs.py#L77-L87